src/shared/components/cancerSummary/CancerSummaryContent.tsx

```diff
@@ -142,6 +154,22 @@ export class CancerSummaryContent extends React.Component<ICancerSummaryContentP
         }, [] as IBarChartSortedData[]);
     }
 
+    private get orderedLabels() {
```

Comment:
this should be typed as Record<keyof ICancerTypeAlterationPlotData, string> 
this will make sure it has the right properties.  also, this doesn't need to be a class member (getter).  just assign this to a local variable of the module above the class